src/Entity/CommitteeMembership.php

```diff
@@ -44,20 +44,20 @@ class CommitteeMembership
     use EntityIdentityTrait;
 
     /**
-     * The committee UUID.
+     * @var Adherent
      *
-     * @var UuidInterface
-     *
-     * @ORM\Column(type="uuid")
+     * @ORM\ManyToOne(targetEntity="Adherent", inversedBy="memberships")
```

Comment:
Is it absolutely needed to have bidirectional link ?

Reply:
I think yes, it existed before and it's used in `Adherent` 
![image](https://user-images.githubusercontent.com/4582866/40415461-30b486a8-5e7b-11e8-8b01-f79adc7773d5.png)